src/SFA.DAS.EmployerAccounts.Web/Controllers/EmployerAgreementController.cs

```diff
@@ -132,13 +132,20 @@ public async Task<ActionResult> SignAgreement(GetEmployerAgreementRequest reques
         [ValidateAntiForgeryToken]
         public async Task<ActionResult> Sign(string agreementId, string hashedAccountId, int? choice)
         {
-            if (choice != ReviewAgreementLater)
+            if (choice == ReviewAgreementLater)
             {
                 return RedirectToAction(ControllerConstants.IndexActionName, ControllerConstants.EmployerTeamControllerName);
             }
 
             var userInfo = OwinWrapper.GetClaimValue(ControllerConstants.UserRefClaimKeyName);
             var agreement = await _orchestrator.GetById(agreementId, hashedAccountId, userInfo);
+
+            if (choice == null)
```

Comment:
Would it be better to have choice default to 0 so a const can be used to define the state?

Reply:
Isn’t the default value passed for a form field with no value null? I’m not sure zero would work and it’d be inconsistent with all other pages.